_episodes/08-memory.md

```diff
@@ -116,6 +116,27 @@ the more likely the fact is to be remembered.  This builds on our earlier idea
 of mental models - a mental model is a way to facilitate making connections between
 separate facts.
 
+> ## Repetition vs. Reflective Practice
+>
+> The idea that ten thousand hours of practice will make someone an expert in some field
+> is widely known,
+> but reality is much more complex.
+> First,
+> practice is not doing the same thing over and over again:
+> practice is doing similar but subtly different things,
+> getting feedback,
+> and then changing behavior in response to that feedback to get cumulatively better.
+> Doing the same thing over and over again is much more likely to solidify bad habits than perfect performance.
+>
+> Second,
+> a [meta-study in 2014][macnamara-practice] found that
+> "...deliberate practice explained 26% of the variance in performance for games,
+> 21% for music, 18% for sports, 4% for education, and less than 1% for professions."
+> One explanation for this variation is that
+> deliberate practice works best when the rules are very stable,
+> but is much less effective when they are fluid or nonexistent.
```

Comment:
Not sure I get this. Are rules for education fluid/non-existent, while they are stable for games and music?